rs/tests/driver/src/driver/vector_vm.rs

```diff
@@ -280,11 +280,18 @@ fn emit_kibana_url_event(log: &slog::Logger, network_name: &str) {
         url: String,
     }
 
+    let now = chrono::Utc::now();
+
+    let before = now - chrono::Duration::hours(1);
```

Comment:
To be even more precise you could consider setting `before` to `now()` as the first step in [make_plan()](https://sourcegraph.com/github.com/dfinity/ic@381c7e93ea1a220692e1c607ed541aa9dfcd0c51/-/blob/rs/tests/driver/src/driver/group.rs?L548) and then passing it via `vector_vm.sync_with_vector()` to `emit_kibana_url_event()`. That way the start time is set to the beginning of the test which is kind of nice.